test/release/examples/primers/parIters.chpl

```diff
@@ -1,67 +1,68 @@
 /*
-  This example shows how to use parallel iterators in Chapel.
+  Parallel Iterators Primer
+   
+  This primer shows how to use parallel iterators in Chapel.
   Leader-follower iterators are used to implement zippered
   forall loops in Chapel over data structures or iterators.
   Standalone parallel iterators are used to implement non-zippered
   forall loops when they exist, falling back to leader-follower when
-  standalone is not available. We expect the parallel iterator
-  interface to change and improve over time, so this should be
-  considered a snapshot of their use in the current implementation.
+  standalone is not available. 
 
   For a more thorough introduction to leader-follower iterators,
   refer to our PGAS 2011 paper, "User-Defined Parallel Zippered
```

Comment:
Maybe make this a URL to http://pgas11.rice.edu/papers/ChamberlainEtAl-Chapel-Iterators-PGAS11.pdf ?

Use the style that allows you to put the URL at the bottom of the file, see [here](http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets)